src/client/conn.rs

```diff
@@ -462,6 +464,24 @@ impl Builder {
         self.h1_read_buf_exact_size = sz;
         self
     }
+
+    /// Set the maximum buffer size for the connection.
+    ///
+    /// Default is ~400kb.
+    ///
+    /// # Panics
+    ///
+    /// The minimum value allowed is 8192. This method panics if the passed `max` is less than the minimum.
+    pub fn max_buf_size(&mut self, max: usize) -> &mut Self {
```

Comment:
Since it's HTTP1 only, perhaps the method should be prefixed with `http1_`, like the other builder methods in `client/mod.rs`.